landing-page/content/common/roadmap.md

```diff
@@ -22,28 +22,29 @@ disableSidebar: true
 
 # Roadmap Overview
 
-This roadmap outlines projects that the Iceberg community is working on, their priority, and a rough size estimate.
-This is based on the latest [community priority discussion](https://lists.apache.org/thread.html/r84e80216c259c81f824c6971504c321cd8c785774c489d52d4fc123f%40%3Cdev.iceberg.apache.org%3E).
+This roadmap outlines projects that the Iceberg community is working on.
 Each high-level item links to a Github project board that tracks the current status.
 Related design docs will be linked on the planning boards.
 
-# Priority 1
+# Iceberg Core / API
```

Comment:
I think this headline is rather misleading, because most work includes support for Spark/Flink/...

Reply:
Lol is there a better term for non-spec?

Perhaps it also makes sense to make a client section when we add to and rust projects too

Reply:
What about

General

Spec

Client

?